setup.py

```diff
@@ -173,90 +206,99 @@ def visit((prefix, strip, found), dirname, names):
     destination = os.path.join(prefix, dirname[strip:])
     found.append((destination, files))
 
+
 def isbad(name):
     """ Whether name should not be installed """
     return (name.startswith('.') or
             name.startswith('#') or
             name.endswith('.pickle') or
             name == 'CVS')
 
+
 def isgood(name):
     """ Whether name should be installed """
     if not isbad(name):
-        if name.endswith('.py') or name.endswith('.json') or name.endswith('.tar'):
+        if  name.endswith('.py')   or \
+            name.endswith('.json') or \
+            name.endswith('.tar'):
             return True
     return False
 
-#-----------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+#
+class RunTwine(Command):
+    user_options = []
+    def initialize_options (self) : pass
+    def finalize_options   (self) : pass
+    def run (self) :
+        out,  err, ret = ru.sh_callout('python setup.py sdist upload -r pypi')
+        raise SystemExit(ret)
+
+
+# ------------------------------------------------------------------------------
 #
 if  sys.hexversion < 0x02060000 or sys.hexversion >= 0x03000000:
-    raise RuntimeError("SETUP ERROR: radical.entk requires Python 2.6 or higher")
+    raise RuntimeError("SETUP ERROR: %s requires Python 2.6 or higher" % name)
 
-# short_version = 0.6
 
+# -------------------------------------------------------------------------------
 setup_args = {
-    'name'             : 'radical.entk',
-    'version'          : version,
-    'description'      : "Radical Ensemble Toolkit.",
-    #'long_description' : (read('README.md') + '\n\n' + read('CHANGES.md')),
-    'author'           : 'RADICAL Group at Rutgers University',
-    'author_email'     : 'vivek.balasubramanian@rutgers.edu',
-    'maintainer'       : "Vivek Balasubramanian",
-    'maintainer_email' : 'vivek.balasubramanian@rutgers.edu',
-    'url'              : 'https://github.com/radical-cybertools/radical.entk',
-    'license'          : 'MIT',
-    'keywords'         : "ensemble workflow execution",
-    'classifiers'      :  [
+    'name'               : name,
+    'version'            : version,
+    'description'        : "Radical Ensemble Toolkit.",
+    'long_description'   : (read('README.md') + '\n\n' + read('CHANGES.md')),
+    'author'             : 'RADICAL Group at Rutgers University',
+    'author_email'       : 'radical@rutgers.edu',
+    'maintainer'         : 'The RADICAL Group',
+    'maintainer_email'   : 'radical@rutgers.edu',
```

Comment:
Does this email exist?

Reply:
Good point - I thought so, but we can also use a different one, like one of our lists.  What I do know is that this specific email has a finite lifetime :-)

Reply:
Makes sense. Which one should we use? I would think maintainer email should be of an individual right or a group email (not rct-devel or something).